moto/athena/models.py

```diff
@@ -174,8 +175,23 @@ def start_query_execution(
         return execution.id
 
     def get_execution(self, exec_id: str) -> Execution:
+        warnings.warn(
```

Comment:
This is an internal API, as far as we're concerned, so we can just rename this method without throwing any warnings.